src/main/resources/application.yaml

```diff
@@ -252,7 +252,7 @@ billPay:
     billStatus: /transferRequests/{{correlationId}}
 
 awaitly:
-  maxWaitTime: 25
+  maxWaitTime: 120
```

Comment:
I am accepting this as the awaitly time out is a max wait time and this doesn't affect tests that complete early.